dashboard/app/jobs/aichat_request_chat_completion_job.rb

```diff
@@ -63,7 +63,7 @@ def perform(request:, locale:)
 
   private def get_execution_status_and_response(request, locale)
     # Moderate user input for toxicity.
-    user_toxicity = AichatSafetyHelper.find_toxicity(request.new_message['chatMessageText'], request.level_id)
+    user_toxicity = AichatSafetyHelper.find_toxicity(request.new_message['chatMessageText'], request.level_id, 'User')
```

Comment:
I wonder if putting a shared constant in for `User` and `Assistant` is worth it? like we have https://github.com/code-dot-org/code-dot-org/blob/3e353b7df632fb49991bf3381a1fab5ef74815a8/apps/src/aiComponentLibrary/chatMessage/types.ts#L2 for the frontend. I don't feel strongly, though!